spec/shr_core_medication.txt

```diff
@@ -76,7 +76,8 @@ Concept:		SCT#416608005 "Drug therapy (procedure)"  // TODO: investigate whether
 Description:	"A record of the use of a medication. 
 Although FHIR supports the asserter (information source) and date asserted, it does not have a place for the author (who created and is responsible for the statement) and recorder (who entered the statement). 
 The treatment of 'not taken' has changed from DSTU2 and STU3 to R4. In R4, status and statusReason are used to indicate medications not taken. For upward compatibility, the 'not taken' attributes are profiled out.
-ReasonCode is a choice of CodeableConcept or ref(Condition) in DSTU2, and limited to one reason (pick one type). In STU3 and R4, there can be multiple reason codes, and multiple reason references, simultaneously."
+ReasonCode is a choice of CodeableConcept or ref(Condition) in DSTU2, and limited to one reason (pick one type). In STU3 and R4, there can be multiple reason codes, and multiple reason references, simultaneously.
+mCODE model gives preference to representing medications using the National Library of Medicine (NLM) RxNorm terminology - a coding standard established by the Office of the National Coordinator (ONC) for the exchange of drugs. RxNorm however is restricted to FDA-approved drugs and does not include clinical trial drugs.mCODE allows for the inclusion of other coding systems like the NCI Thesaurus (NCIT) to represent clinical trial oncology drugs."
```

Comment:
The description of a class should not reference mCODE as the class may be used in the context of other IGs. Mark is already addressing this in other element descriptions.